folium/features.py

```diff
@@ -73,8 +76,9 @@ def __init__(self, url, name=None, layers=None, styles=None, format=None,
                     styles: '{{ this.styles }}',
                     format: '{{ this.format }}',
                     transparent: {{ this.transparent.__str__().lower() }},
-                    version: '{{ this.version }}',
-                    {% if this.attribution %} attribution: '{{this.attribution}}'{% endif %}
+                    version: '{{ this.version }}'
+                    {% if this.attribution %}, attribution: '{{this.attribution}}'{% endif %}
+                    {% for param in this.extra_params %}, {{param.0}}: '{{param.1}}'{% endfor %}
```

Comment:
@bielfrontera where do you stand on adding those parameters explicitly instead of looping over them? We also need to update the docs to describe them. At least the most important subset you expect to use.